src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockController.java

```diff
@@ -0,0 +1,15 @@
+package use_cases.display_availability_timeblocks;
+
+import java.time.LocalTime;
+
+public class DisplayAvailabilityTimeBlockController {
+    private final DisplayAvailabilityTimeBlockInteractor displayAvailabilityTimeBlockInteractor;
```

Comment:
This is perfect! We always want to make sure that our clean architecture-related classes exclusively hold _private_ instance attributes; this will help keep our code clean.